packages/template/package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "postbuild": "cross-env NODE_OPTIONS='--experimental-json-modules --experimental-modules' node ./scripts/search.mjs",
+    "postbuild": "cross-env NODE_OPTIONS=\"--experimental-json-modules --experimental-modules\" node ./scripts/search.mjs",
```

Sorry, I took the freedom to fix this one as well before the review (it doesn't directly relate to the PR title, but it still fixes a Windows-only problem) 😅. Apparently, Windows (or CMD maybe?) doesn't like the single quotes, but the double quotes should work on most platforms. But also, I can't help but wonder why not use the simpler form (that doesn't depend on `cross-env`):
```suggestion
    "postbuild": "node --experimental-json-modules --experimental-modules ./scripts/search.mjs",
```
?

---

This was causing another silent fail on Windows with `pnpx flowershow export`. I was wondering why it generated the `.next` output but not the static one, and figured out that this is the root cause.
This was (hopefully) the last issue stopping me from adopting Flowershow for my PKM (there are a few other minor issues, but nothing critical).